src/pages/boundary/downloads/index.tsx

```diff
@@ -24,6 +23,4 @@ export const getStaticProps: GetStaticProps = async () => {
   return generateStaticProps(product)
 }
 
-BoundaryDownloadsPage.layout = CoreDevDotLayout
```

Comment:
Related but less consequential change: all `/{product}/downloads` views already use `CoreDevDotLayout`, via product downloads view, which uses sidebar-sidecar layout under the hood, which uses core dev dot layout under the hood.